streamlit_folium/__init__.py

```diff
@@ -315,6 +339,10 @@ def bounds_to_dict(bounds_list: list[list[float]]) -> dict[str, dict[str, float]
                 idx=idx,
             )
 
+    layer_control_string = None
+    if layer_control is not None:
+        layer_control_string = _get_layer_control_string(layer_control, folium_map)
+
```

Comment:
Would be nice to add this string to the debug info as well

Reply:
done